### DISCUSSION TITLE
Front-end and back-end associated by traceId

### DISCUSSION THREAD

**Discussion by bearxiongzhen** [apache/skywalking#8160] [General]
javascript + java traceID
Front-end and back-end association

**Comment by wu-sheng** [comment]
So? What is the question?

**Comment by bearxiongzhen** [comment]
Does skywalking support the front-end and back-end link tracking function to connect the front-end and the back-end in series

**Reply by wu-sheng** [reply]
Yes, it is there for years. Follow quick start, we have showcase for you.
One command to deploy most features for preview.

**Comment by bearxiongzhen** [comment]
this one?
https://github.com/apache/skywalking-client-js
